modules/xmpp/JingleSessionPC.ts

```diff
@@ -1231,13 +1581,23 @@ export default class JingleSessionPC extends JingleSession {
         this.wasstable = false;
         this.webrtcIceUdpDisable = Boolean(options.webrtcIceUdpDisable);
         this.webrtcIceTcpDisable = Boolean(options.webrtcIceTcpDisable);
-
-        const pcOptions = { disableRtx: options.disableRtx };
+        const pcOptions = {
+            audioQuality: undefined,
+            capScreenshareBitrate: undefined,
+            codecSettings: undefined,
+            disableRtx: options.disableRtx,
+            disableSimulcast: undefined,
+            enableInsertableStreams: undefined,
+            forceTurnRelay: undefined,
+            maxstats: undefined,
+            startSilent: undefined,
+            usesCodecSelectionAPI: undefined,
+            videoQuality: undefined
```

Comment:
Why put all the keys from the start and then override them? you can assign many of them directly no?